tests/integration/tier1/config-file/main.fmf

```diff
@@ -1,5 +1,15 @@
-summary: config file
+summary: Test config file
 
-tier: 0
+description: |
+    Testing the configuration file option. There is need to test the
+    priority of the possible configuration files paths combined with
+    the CLI options. Verify that the correct data were loaded and if applicable
+    a warning is printed.
```

Comment:
```suggestion
    Run Convert2RHEL using a configuration file for sensitive information. 
    Verify that the conversion can be done with different configuration paths, as well, CLI options and to assert that the configuration will take precendence over the CLI.
    Verify that the correct data were loaded and if applicable a warning is printed.
```

Reply:
Thanks @r0x0d I've missed this change, sorry. Updated and rebased